app/models/twitter_user.rb

```diff
@@ -336,16 +338,16 @@ def favoriting(options = {})
   end
 
   def inactive_friends
-    ExTwitter.new.select_inactive_users(friends, authorized: ego_surfing?)
+    ExTwitter::Client.new._extract_inactive_users(friends, authorized: ego_surfing?)
```

Comment:
__[rubocop]__
* Line is too long. [84/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)